### PR TITLE
Quirks for CentraLite 3321-S

### PR DESCRIPTION
**Pull request by roblandry**

### CODE REVIEW

File: zhaquirks/centralite/3321S.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+from zigpy.profiles import PROFILES, zha
+from zigpy.zcl.clusters.general import Basic, Identify,\
+    PollControl, Ota
+from zigpy.zcl.clusters.security import IasZone
+from zhaquirks.centralite import PowerConfigurationCluster
+from zhaquirks.smartthings import SmartThingsAccelCluster
```
Comment (dmulcahey) [inline zhaquirks/centralite/3321S.py]: get this from quirks in zigpy instead and drop the version from this PR

File: zhaquirks/centralite/3321S.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+from zigpy.quirks import CustomDevice
+
+
+DIAGNOSTICS_CLUSTER_ID = 0x0B05  # decimal = 2821
+MANUFACTURER_SPECIFIC_PROFILE_ID = 0xC2DF  # decimal = 49887
+MANUFACTURER_SPECIFIC_CLUSTER_ID = 0xFC0F  # decimal = 64527
+
+class CentraLite3321S(CustomDevice):
+
+    def __init__(self, *args, **kwargs):
```
Comment (dmulcahey) [inline zhaquirks/centralite/3321S.py]: remove this

File: zhaquirks/centralite/3321S.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        super().__init__(*args, **kwargs)
+
+    signature = {
+        #  <SimpleDescriptor endpoint=1 profile=260 device_type=1026
+        #  device_version=0
+        #  input_clusters=[0, 1, 3, 1026, 1280, 32, 2821, 64514]
+        #  output_clusters=[25]>
+        1: {
+            'profile_id': zha.PROFILE_ID,
```
Comment (dmulcahey) [inline zhaquirks/centralite/3321S.py]: we match on man / model now so add them here

File: zhaquirks/centralite/3321S.py (hunk 1 above)
Comment (dmulcahey) [reply zhaquirks/centralite/3321S.py]: scratch that... import from Centralite after you move it in the other PR...

File: zhaquirks/centralite/3321S.py (hunk 2 above)
Comment (roblandry) [reply zhaquirks/centralite/3321S.py]: Done

File: zhaquirks/centralite/3321S.py (hunk 3 above)
Comment (roblandry) [reply zhaquirks/centralite/3321S.py]: Done

**Pull request merged**